docs/excel/excel-add-ins-ranges-advanced.md

```diff
@@ -322,6 +322,37 @@ Excel.run(function (context) {
 
 To ungroup a row or column group, use the [Range.ungroup](/javascript/api/excel/excel.range#ungroup-groupoption-) method. This removes the outermost level from the outline. If multiple groups of the same row or column type are at the same level within the specified range, all of those groups are ungrouped.
 
+## Handle dynamic arrays and spilling (preview)
+
+> [!NOTE]
+> Dynamic array and range spilling APIs are currently in preview. [!INCLUDE [Information about using preview Excel APIs](../includes/using-excel-preview-apis.md)]
+
+Some Excel formulas return [Dynamic arrays](https://support.microsoft.com/office/dynamic-array-formulas-and-spilled-array-behavior-205c6b06-03ba-4151-89a1-87a7eb36e531). These "spill" their results across multiple cells. Your add-in can track the ranges that have been spilled into with the [Range.getSpillingToRange](/javascript/api/excel/excel.range#getspillingtorange--) method. There is also a [*OrNullObject version](excel-add-ins-advanced-concepts.md#ornullobject-methods), `Range.getSpillingToRangeOrNullObject`.
+
+The following sample shows a basic formula that copies the contents of a range into a cell, which spills into neighboring cells. The add-in then logs the range that's been spilled into.
```

Comment:
The "range that's been spilled into" phrasing (here and a few lines prior) feels confusing to me, is that how the docs usually refer to spills? Perhaps... 
```suggestion
The following sample shows a basic formula that copies the contents of a range into a cell, which spills into neighboring cells. The add-in then logs the range containing the spill.
```